index.bs

```diff
@@ -179,8 +190,65 @@ extensible.
 </script>
 ```
 
+## Display Arbitrary Content ## {#example-arbitrary-content}
+
+```html
+<div style="display: none;">
+  <div id="interactive-ui"></div>
+  <div id="non-interactive-ui"></div>
+</div>
+
+<video id="video" src="https://example.com/file.mp4"></video>
+
+<button id="pipButton"></button>
+
+<script>
+  const video = document.getElementById('video');
+  const pipButton = document.getElementById('pipButton');
+  const interactiveUI = document.getElementById('interactive-ui');
+  const nonInteractiveUI = document.getElementById('non-interactive-ui');
+
+  pipButton.addEventListener('click', () => {
+    interactiveUI.requestPictureInPicture({ interactive: true })
+      .catch(e =>
+          return nonInteractiveUI.requestPictureInPicture({ interactive: false }))
+      .then(e =>
+          e.pictureInPictureWindow.document.body.appendChild(video));
+  });
+</script>
+```
+
 # Concepts # {#concepts}
 
+## Check Picture-in-Picture Is Allowed ## {#pip-allowed}
+
+When the <dfn>check Picture-in-Picture is allowed algorithm</dfn> with
+|element| and |userActivationRequired| is invoked, the user agent MUST run
+the following steps:
+
+1. If <a>Picture-in-Picture support</a> is `false`, throw a
+    {{NotSupportedError}} and abort these steps.
+2. If the document is not allowed to use the <a>policy-controlled feature</a>
+    named `"picture-in-picture"`, throw a {{SecurityError}} and abort these
+    steps.
+3. If |userActivationRequired| is `true` and the algorithm is not
+    <a>triggered by user activation</a>, throw a {{NotAllowedError}} and
+    abort these steps.
+
+## Update Picture-in-Picture Element ## {#update-element}
+
+When the <dfn>update Picture-in-Picture element algorithm</dfn> with |element|
+and |document| is invoked, the user agent MUST run the following steps:
+
+1. Set {{pictureInPictureElement}} to |element|.
+2. Let <dfn>Picture-in-Picture window</dfn> be a new instance of
+    {{PictureInPictureWindow}} associated with {{pictureInPictureElement}} with
+    a reference to |document|.
```

Comment:
It isn't the case for video. Shall we do this separately?

Reply:
Done.